### PR TITLE
Doc on How to Add Objects

### PR DESCRIPTION
**Pull request by ThomasSchellenbergNextCentury**
My first pass at this doc.

I'm not sure where exactly this should go -- let me know if you have a suggestion.

Step 3 was taken from our README [here](https://github.com/NextCenturyCorporation/ai2thor/#modifying-a-prefab-to-work-in-the-mcs-scene), so I'll replace that section with a link to this doc instead.

I'm not sure if we should keep the Troubleshooting section, given what Victoria said this morning about contacting our team.

### CODE REVIEW

File: ADDING_MODELS.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+# Adding 3D Models
+
+Here are the steps to add your own 3D models into the MCS simulation environment.
```
Comment (deanwetherby) [inline ADDING_MODELS.md:3]: This is essentially the TLDR with the glorious details below. Might be good to state that even though the heading imply but it's subtle.

File: ADDING_MODELS.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+- Successfully integrating existing objects that are part of the original AI2-THOR project but not yet available in the MCS simulation environment may necessitate making MCS-specific adjustments to those objects.
+- This is a manually-intensive procedure that requires using the Unity Editor. Since human perception is needed for indentifying the correct pixels to position bounding boxes around and/or inside specific parts of the 3D model, we're not currently planning on making this procedure automated or dynamic. This is exactly how the MCS Core Team adds models into the simulation environment -- we have no secret shortcuts here.
+
+## 1. Download and Open MCS in the Unity Editor
+
+Clone the [MCS fork of AI2-THOR](https://github.com/NextCenturyCorporation/ai2thor) and follow the project's [setup instructions](https://github.com/NextCenturyCorporation/ai2thor#setup) to download and install the correct version of the Unity Editor and use it to load the MCS Unity project.
+
+## 2. Import Your Models into Unity and Save Them as Prefabs
+
+(Continuing from Step 1)
```
Comment (deanwetherby) [inline ADDING_MODELS.md:27]: I don't think this it is necessary to state the continuation.

File: ADDING_MODELS.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+2B. Navigate to the folder containing your model using the Project window of the Unity Editor. You should see your model there as a thumbnail.
+
+2C. Click and drag your model's thumbnail into the Scene window. Your model should appear in the empty scene, and the name of your model's name should appear in the list in the Hierarchy window.
+
+2D. Click and drag your model's name from the list in the Hierarchy window into your folder in the Project window. You should see a "Create Prefab" popup appear. Click the "Original Prefab" button to create a Unity Prefab file (".prefab") for your model. Please see the Unity Manual for more information about Prefabs.
+
+## 3. Update Your Prefabs in Unity
+
+(Continuing from Step 2)
```
Comment (deanwetherby) [inline ADDING_MODELS.md:39]: I don't think this it is necessary to state the continuation.

File: ADDING_MODELS.md [hunk 4]
```diff
@@ -115,32 +115,73 @@ Clone the [MCS fork of AI2-THOR](https://github.com/NextCenturyCorporation/ai2th
...
+7D. To add new objects into passive agent scenes, in [agent_scene_pair_json_converter.py](https://github.com/NextCenturyCorporation/mcs-scene-generator/blob/master/agent_scene_pair_json_converter.py), see the `AGENT_OBJECT_CONFIG_LIST` and `GOAL_OBJECT_CONFIG_LIST` variables. Add a new ObjectConfig to either one or both of the two arrays, as needed.
 
 ## 8. Submit a Pull Request
 
 Optionally, if you would like to contribute your objects to the public MCS simulation environment (in our GitHub source code repository) for other teams to use, and have legal permission to share your models, then you may submit a Pull Request to the MCS fork of AI2-THOR containing your new Prefabs and updates to the object registry file.
 
 ## Troubleshooting
 
-If you have any questions about the procedure, please contact the MCS Core Team via a GitHub Issue, the MCS Slack, or email: mcs-ta2@machinecommonsense.com
+If you have any questions about the procedure, please contact the MCS Core Team by opening a new GitHub Issue on this repository, posting on the #ta2-api MCS Slack channel, or emailing: mcs-ta2@machinecommonsense.com
```
Comment (deanwetherby) [inline ADDING_MODELS.md:187]: It's ok to leave the troubleshooting section. I don't mind providing clarification on the instructions/steps themselves but we should be explicit that we can't help each team if they run into issues beyond that.

**Review by deanwetherby — COMMENTED** [review]
There's so much information in this doc. Thanks for putting that together. Right now it's a big blob of text. I think some screenshots would go a long way.

**Comment by deanwetherby** [comment]
I think this markdown file should go in the machine_common_sense folder along with DEV.md. We can link/reference from the project README

**Review by brianpippin — APPROVED** [review]
Everything seems like what it is suppose to be, I didn't notice anything glaring, but reading this reminds me how intensive this process is.

**Review by rartiss55 — APPROVED** [review]
Thanks for putting this together, I think its also useful for our internal team.

**Pull request merged**